packages/drivers/odsp-driver/src/odspFluidFileLink.ts

```diff
@@ -128,7 +129,10 @@ export function storeLocatorInOdspUrl(url: URL, locator: OdspFluidDataStoreLocat
  * @param url - ODSP url representing Fluid file link
  * @returns object representing Fluid data store location in ODSP terms
  */
-export function getLocatorFromOdspUrl(url: URL): OdspFluidDataStoreLocator | undefined {
+export function getLocatorFromOdspUrl(
+	url: URL,
+	requireFluidSignature?: boolean,
```

Comment:
Should not this param be set to true by default to retain backward compatibility?

Reply:
Is it redundant to set the default parameter to true for both `getLocatorFromOdspUrl` and `decodeOdspFluidDataStoreLocator`? I can add it here anyway